lib/api/api_auth_webauthn.go

```diff
@@ -162,11 +156,16 @@ func (u webauthnLibUser) WebAuthnCredentials() []webauthnLib.Credential {
 			transports[i] = webauthnProtocol.AuthenticatorTransport(t)
 		}
 
+		credentialVolState, ok := webauthnVolState.Credentials[cred.ID]
+		if !ok || credentialVolState == nil {
```

Comment:
Dumb question in case it trigger some helpful realisation of an issue/improvement somewhere, otherwise just ignore: Why would there be a nil entry present - I'd expect whatever adds a nil entry to not do that or delete the existing entry instead.